extension/chrome/elements/compose-modules/compose-render-module.ts

```diff
@@ -122,6 +131,14 @@ export class ComposeRenderModule extends ViewModule<ComposeView> {
     this.view.S.cached('compose_table').css('display', 'none');
     this.view.S.cached('reply_msg_successful').find('div.replied_from').text(this.view.senderModule.getSender());
     this.view.S.cached('reply_msg_successful').find('div.replied_to span').text(msg.headers.To.replace(/,/g, ', '));
+    if (ccHeaders.length > 0) {
+      this.view.S.cached('reply_msg_successful').find('div.replied_cc span').text(ccHeaders.join(','));
```

Comment:
```diff
- this.view.S.cached('reply_msg_successful').find('div.replied_cc span').text(ccHeaders.join(','));
+ this.view.S.cached('reply_msg_successful').find('div.replied_cc span').text(ccHeaders.join(', '));
```

Reply:
I have done removing other unnecessary code and I use the `msg.recipients` instead, Thanks!

Reply:
this was the request was about the missing space symbol:

<img width="77" alt="CleanShot 2021-09-03 at 13 05 36@2x" src="https://user-images.githubusercontent.com/6059356/131988490-a0c2fde8-1c2d-4918-a8d0-b7e9ced31cc0.png">

Reply:
@martgil ping